hyperwallet/api.py

```diff
@@ -2791,4 +2792,41 @@ def createTransferSpendBackRefund(self,
             params
         )
 
-        return TransferRefunds(response)
\ No newline at end of file
+        return TransferRefunds(response)
+
+    '''
+
+    Create User Transition
+
+    '''
+
+    def createUserStatusTransition(self,
+                                   userToken=None,
+                             data=None):
+        '''
+        Create User Transition.
+
+        :param data:
+            A dictionary containing User Create UserToken information. **REQUIRED**
+        :param data:
+            A dictionary containing User Create Data information. **REQUIRED**
+        :returns:
+            CreateUserStatusTransition.
+        '''
+
+        if not userToken:
+            raise HyperwalletException('userToken is required')
+
+        if not data:
+            raise HyperwalletException('data is required')
+
+        response = self.apiClient.doPost(
```

Comment:
Pls make it more readable by having the parameters in the same line instead of different lines. Fold the lines only when they are too long.

Reply:
as per your comments has been resolved